CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Setup the environment
+
+1. Clone the repo: `git clone git@github.com:python-lsp/python-lsp-server.git`
+2. Create the virtual environment: `python3 -m venv .venv`
+3. Activate: `source .venv/bin/activate`
+4. Install an editable installation: `pip3 install -e .`
+    - This will ensure you'll see your edits immediately without reinstalling the project
+
+## Configure your editor
+
+In Sublime Text 4, open LSP-pylsp settings and change the path to the `pylsp` command:
```

Comment:
I don't know if mentioning one preferred editor over others is a good idea because the setup and development workflow is completely different for each of them. I mean, I could write a guide for Spyder as well (which would be useful for our interested contributors) and it'd be another thing entirely.

So, perhaps having a contributing guide per editor would be better?

Reply:
I understand, I initially mentioned ST4 because making it use a custom pylsp path was very difficult, so I felt it was important. But yes, there's too many editors and IDEs to cover that it might not be a good idea to have it in the contributing guide.

> So, perhaps having a contributing guide per editor would be better?

That sounds like a good idea.

Reply:
>> So, perhaps having a contributing guide per editor would be better?

>That sounds like a good idea.

Or perhaps none at all? Surely, those who are able and willing to contribute will be able to configure their editor already.

Reply:
Removed the ST config in favor of a generic point of configuring the editor.